submissions/mwansaMwansa/database2/transportation.sql

```diff
@@ -1,30 +1,32 @@
 
 CREATE DATABASE transportation;
 
-CREATE TABLE registered_passengers (
+CREATE TABLE passengers_details (
 	id INT NOT NULL auto_increment,
     full_name VARCHAR(250) NOT NULL,
     sex VARCHAR(8) NOT NULL,
-    age INT NULL,
-    primary key(id)
+    age VARCHAR NULL,
```

Comment:
age Should remain INT.
With VARCHAR (3), 0I5 would be accepted though it is wrong and different from 015